crates/solver/src/driver.rs

```diff
@@ -383,7 +383,7 @@ impl Driver {
             auction: competition_auction,
             solutions: rated_settlements
                 .iter()
-                .map(|(solver, rated_settlement, _)| SolverSettlement {
+                .map(|(solver, rated_settlement)| SolverSettlement {
```

Comment:
Actually kind of surprised that this really wasn't used anywhere except for logging... Nice cleanup 🧹